From bb25aac568359efbfb68e52dd8d9ee85e65d754b Mon Sep 17 00:00:00 2001
From: Haelwenn Monnier <lanodan@users.noreply.github.com>
Date: Thu, 25 Oct 2018 09:10:16 +0200
Subject: [PATCH] xim/main.cpp: FIx build against clang

main.cpp:68:55: error: invalid suffix on literal; C++11 requires a space between literal and identifier
---
 xim/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xim/main.cpp b/xim/main.cpp
index a0b0beceb..c5818ca4a 100644
--- a/xim/main.cpp
+++ b/xim/main.cpp
@@ -65,7 +65,7 @@ int g_option_mask;
 int scr_width, scr_height;
 int host_byte_order;
 
-#define VERSION_NAME "uim-xim under the way! Version "PACKAGE_VERSION"\n"
+#define VERSION_NAME "uim-xim under the way! Version " PACKAGE_VERSION "\n"
 const char *version_name=VERSION_NAME;
 const char *usage=
 "--help , --version :Show usage or version\n"
